docs/logging-and-managing-experiment-results/logging-experiment-data.rst

```diff
@@ -250,6 +250,10 @@ You will have all sources in the |source-code| section of the experiment. Neptun
 
 |example-code-snapshot|
 
+.. warning::
+
+    When using pattern expansion, such as ``'*.py'``, make sure that your expression does not log too many files or non-source code files. For example using ``'*'`` as a pattern will upload all files and directories from the cwd. It may result in logging files that you did not want to upload and cluttering your storage.
```

Comment:
When using pattern expansion, such as ``'*.py'``, make sure that your expression does not log too many files or non-source code files. For example, using ``'*'`` as a pattern will upload all files and directories from the cwd. It may result in logging files that you did not want to upload and to clutter your storage.

Reply:
ok